tests/IceRpc.Quic.Tests/Transports/QuicIdleTimeoutTests.cs

```diff
@@ -60,14 +60,56 @@ public async Task Quic_connection_idle_after_idle_timeout([Values]bool configure
         // Act / Assert
         var startTime = TimeSpan.FromMilliseconds(Environment.TickCount64);
 
-        Assert.That(readResult.Buffer.ToArray(), Is.EqualTo(data));
-
         Assert.That(
             async () => await sut.Local.Input.ReadAsync().AsTask(),
             Throws.InstanceOf<IceRpcException>().With.Property("IceRpcError").EqualTo(IceRpcError.ConnectionIdle));
 
         Assert.That(
             TimeSpan.FromMilliseconds(Environment.TickCount64) - startTime,
             Is.GreaterThan(TimeSpan.FromMilliseconds(490)));
+
+        Assert.That(readResult.Buffer.ToArray(), Is.EqualTo(data));
+    }
+
+    /// <summary>Verifies the QUIC connection is kept alive by the keep alive interval.</summary>
+    [Test]
+    public async Task Quic_connection_kept_alive_by_keep_alive_interval()
+    {
+        // Arrange
+        var services = new ServiceCollection().AddQuicTest();
+
+        services.AddOptions<QuicServerTransportOptions>("server").Configure(
+            options => options.IdleTimeout = TimeSpan.FromMilliseconds(500));
+
+        services.AddOptions<QuicClientTransportOptions>("client").Configure(
+            options => options.KeepAliveInterval = TimeSpan.FromMilliseconds(250));
+
+        await using ServiceProvider provider = services.BuildServiceProvider(validateScopes: true);
+
+        var clientServerConnection = provider.GetRequiredService<ClientServerMultiplexedConnection>();
+        await clientServerConnection.AcceptAndConnectAsync();
+        using var sut = await clientServerConnection.CreateAndAcceptStreamAsync(bidirectional: true);
+
+        // Simulate a request
+        var data = new byte[] { 0x1, 0x2, 0x3 };
+        await sut.Local.Output.WriteAsync(data);
+        ReadResult readResult = await sut.Remote.Input.ReadAsync();
+
+        // Act/Assert
+
+        // Without the keep-alive interval, the idle timer aborts the connection after 500ms.
+        using var cts = new CancellationTokenSource(TimeSpan.FromSeconds(2));
+
+#if NET9_0_OR_GREATER
```

Comment:
This assumes the unit tests are built with the same version of .NET as the IceRpc.Transport.Quic assembly (a safe assumption).